bundles/AdminBundle/src/Controller/Admin/SettingsController.php

```diff
@@ -1539,6 +1539,9 @@ public function websiteSettingsAction(Request $request): JsonResponse
         return $this->adminJson(['success' => false]);
     }
 
+    /**
+     * @return array{id: ?int, name: string, language: string, type: string, data: ?string, siteId: ?int, creationDate: ?int, modificationDate: ?int}
```

Comment:
```suggestion
     * @return array{id: ?int, name: string, language: string, type: string, data: mixed, siteId: ?int, creationDate: ?int, modificationDate: ?int}
```
Since $item->getData() https://github.com/pimcore/pimcore/blob/ae07a34d3eed0fafadb1b58a5a59d3f5440af135/bundles/AdminBundle/src/Controller/Admin/SettingsController.php#L1569 can return mixed